SuperPower-uC/README.md

```diff
@@ -33,3 +44,61 @@ Examples:
 
 `Fixed missing ESP32 Module`
 
+* a git gui tool with timeline view is strongly recommended
+* each contributor shall pro actively detect conflicts he creates or created by other branches and get in touch to define a conflict resolution strategy
+* key files such as readme and information such as BOM and requirements should be checked by all contributors before merge
+
+## Project phases
+* Requirements
+* Components selection
+* Schematics design
+* routing
+* testing and validation
+
+## Guidelines for project progress
+* Each step of the project progress shall be validated in a group meeting
+* After validation, each change in the step (requirements, component) shall be agreed with the Team and not with two reviewers only and PR
+* Each feature (mcu / cut_off / charging / low power ...) shall have a maturity level
+  * 1 ) clear concept
+  * 2 ) design ready
+  * 3 ) functional
+  * 4 ) stable and bug free
+* The maturity level is decided by the Team during a meeting
+
+# Requirements
+With the help of a lot participants we collected a list of requirements for this project.
+
+## Filtered Requirements
+[Requirements ESP32 Boards shared spreadsheet](https://docs.google.com/spreadsheets/d/1Awvs3CumJ9qQSrG_uEvt4s6nZwliIjnMdfPibDWA3kU/edit#gid=1880015614&fvid=1986096210)
+
```

Comment:
It could be good to add a list with the possible categories and their description. (Not mandatory)

Reply:
I agree, I think I will add that in a next version, I would also like to add a status for each requirement and maybe a maturity level such as not only accepted / rejected, but also implemented / tested and so on. In this PR I only performed a reformatting.